operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -264,16 +264,15 @@ Activities:
         * Sizing recommendations
   * Product areas: Functional Overview and Roadmap
     * End user experience \(Desktop, Web, and Mobile Applications\) \(Eric Sethna\)
-    * Cloud and Admin Experience \(Adam Clarkson\)
-    * App Marketplace and Developer Toolkit \(Aaron Rothschild\)
-    * Platform \(Cloud Platform features, Self-Managed Platform features\) \(Katie Wiersgalla\)
-    * Product-centric GTM \(Customer Portal, Growth Product\) \(Jason Blais\)
-    * Workflows \(Ian Tao\)
+    * Cloud and Admin Experience \(Katie Wiersgalla\)
```

Comment:
@wiersgallak do you have a link to the summarized version of product areas of ownership? I found the detailed spreadsheet at https://docs.google.com/spreadsheets/d/1-EUPdTpOxNx4bwf-312Pgqis7xSObK00c-KYwPXr2Y0/edit#gid=0, but I thought there was a summarized version we prepared for sales/CS a few weeks ago.

(The AORs may need updating with the recent updates)